django-data/image/validation/tasks.py

```diff
@@ -69,7 +71,9 @@ def send_message(self, status, submission_obj):
             send_message_to_websocket(
                 {
                     'message': STATUSES.get_value_display(status),
-                    'notification_message': submission_obj.message
+                    'notification_message': submission_obj.message,
```

Comment:
If you see the code, I moved `send_message_to_websocket` under the `ValidateTask.send_message` method since it was called everytime the same

Reply:
And if you see its test method in `valdiation.tests.test_tasks`, I modelled all the websocket tests in the`setUp` method and they are checked in `check_async_called` method. This will simplify the method definition with decorators. You will see also that I have also raised the number of animal and samples in tests, this was introduced in pr #26 . Some values are determined on the fly instead than setting them, I hope to avoid issues if I need to add a more sample, for example

Reply:
👍